draft-ietf-quic-v2.md

```diff
@@ -118,9 +118,11 @@ The values below were chosen randomly.
 The salt used to derive Initial keys in {{Section 5.2 of QUIC-TLS}} changes to:
 
 ~~~
-initial_salt = 0xa707c203a59b47184a1d62ca570406ea7ae3e5d3
+initial_salt = 0x3a785fb8f3009837546c9e8570b6bac843e3e142
 ~~~
 
+This is the first 20 bytes of the sha256sum of "QUICv2 salt."
```

Comment:
Same as above.  Note that this assumes that echo is producing '\n' and not '\r\n' or any of the alternatives, so you might want to use `echo -n` instead.  Also, the period inside the quotes is definitely misleading.